src/bun.js/javascript.zig

```diff
@@ -3334,9 +3334,7 @@ pub const ModuleLoader = struct {
         JSC.markBinding(@src());
         var log = logger.Log.init(jsc_vm.bundler.allocator);
         defer log.deinit();
-        var _specifier = specifier_ptr.toSlice(jsc_vm.allocator);
-        defer _specifier.deinit();
-        var specifier = normalizeSpecifier(jsc_vm, _specifier.slice());
+        var specifier = normalizeSpecifier(jsc_vm, specifier_ptr.slice());
```

Comment:
This will be invalid memory when the specifier_ptr is a UTF-16 string

The specific example seems to be a latin1 string.